test/blackbox-tests/test-cases/dune-ppx-driver-system/driver-tests/dune

```diff
@@ -85,4 +85,7 @@
 (library
  (name test_ppx_staged)
  (modules test_ppx_staged)
- (preprocess (staged_pps -arg1 driver_print_tool -arg2 -- -foo bar)))
+ (preprocess 
+   (staged_pps -arg1 driver_print_tool -arg2 -arg3=%{env:SOME_UNDEFINED_VARIABLE=undefined} 
+            -- -foo bar %{env:SOME_UNDEFINED_VARIABLE=undefined})
```

Comment:
Note to the reviewer: I struggled to find a variable that is actually ... a constant in (almost) all context; that's why I used this "trick" ; the test will fail on machine that define a variable called "SOME_UNDEFINED_VARIABLE".

Reply:
Ack. I wondered about this actually. To avoid the last pitfall, you can add the following stanza in this `dune` file:

```
(env (_ (env-vars (X foo))))
```